File: Overloading.java
import java.util.*;
class Student{
    void display()
    {
        System.out.println("I am a student");
    }
    void display(String name)
    {
        System.out.println("I am a student of "+name);
    }
}
class Main{
    public static void main(String[] args) {
        Student obj=new Student();
        Scanner s=new Scanner(System.in);
        obj.display();
        System.out.print("Enter your college name : ");
        String name=s.nextLine();
        obj.display(name);
        s.close();
    }
}

File: Overloading.cpp
#include<iostream>
using namespace std;
class Student{
    public:
    void display()
    {
        cout<<"I am a student"<<endl;
    }
    void display(string college)
    {
        cout<<"I am a student of "<<college<<endl;
    }
};
int main()
{
    Student obj;
    obj.display();
    string name;
    cout<<"Enter your college name : ";
    cin>>name;
    obj.display(name);
}